source/app/controllers/index.rb

```diff
@@ -1,3 +1,38 @@
 get '/' do
   erb :index
 end
+
+get '/game' do
+  pokemon = Pokemon.all.pluck(:name)
+  erb :game, locals: {pokemon: pokemon}
+end
+
+post '/game' do
+  # winner = Logic.new(params[:pokemon1], params[:pokemon2])
+  # loser =
+  p params
+  player1 = Player.find_by(id: 1)
+  player2 = Player.find_by(id: 2)
+  choice1 = Pokemon.find_by(name: params[:pokemon1])
+  choice2 = Pokemon.find_by(name: params[:pokemon2])
```

Comment:
Not vital, but some error checking might be nice here.

    return [400, "no such pokemon: #{params[:pokemon1]}"] unless choice1